tests/gptq/test_quantization.py

```diff
@@ -300,6 +303,16 @@ class GPTQTestNoBlockCaching(GPTQTest):
     EXPECTED_OUTPUTS.add("Hello my name is John, I am a student in the University of")
 
 
+class GPTQTestInsideLayerModules(GPTQTest):
+    # all layers are quantized apart from self_attention.dense
+    inside_layer_modules = [["self_attention.query_key_value"], ["mlp.dense_h_to_4h"], ["mlp.dense_4h_to_h"]]
```

Comment:
Couldn't it be
```suggestion
    inside_layer_modules = ["self_attention.query_key_value", "mlp.dense_h_to_4h", "mlp.dense_4h_to_h"]
```

Reply:
no since we want to quantize sequentially the layers. First update `self_attention.query_key_value`, then `mlp.dense_h_to_4h` and finally `mlp.dense_4h_to_h`. Using a list is useful when we want for example to update the q,k,v layers at the same time since they are independent. However, for `mlp.dense_h_to_4h`, we want to have q,k,v quantized before.